app/src/main/res/layout/item_content_cart.xml

```diff
@@ -29,17 +31,18 @@
                 android:layout_margin="@dimen/dp_5"
                 android:scaleType="centerCrop"
                 app:imageUrl="@{viewModel.productImage}"
+                tools:ignore="ContentDescription"
                 />
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:layout_marginLeft="@dimen/dp_10"
+                android:layout_marginStart="@dimen/dp_5"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_marginLeft="@dimen/dp_5"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.